generator/templates/Default/src/store/modules/Error.js

```diff
@@ -0,0 +1,38 @@
+import {ADD, CLEAR, FIND, REMOVE} from '../constants.js';
```

Comment:
Ik heb je dit al eerder zien doen? Maar waarom eigenlijk? het is niets zeggend..

- CLEAR : wat clear je nu? de gehele store?
- ADD: wat add je, bij de errors is dat idd een error maar in vele andere stores is deze onbruikbaar
- REMOVE: hetzelfde /\
- FIND: hetzelfde /\

En wat als je meerdere mutations/getters hebt dan werkt dat ineens heel anders en niet via een const?

Reply:
This is to remove redundancy, you don't have to always use those. I thought these were just the most conventional used keywords for the stores we've used in the past.

e.g. Profile store has a `clearProfile`, which is redundant, and sometimes a `fetchProfile` too, which in this case is not a constant, but maybe we should make one too, because I've seen it happening in some projects.

Reply:
Maar maakt dat het helderder?
Dat wanneer je dus een action of getter gaat mappen je ook nog eens die constants moet inladen om dat voor elkaar te krijgen bij een aantal van de methods.

als ik mag kiezen zou ik dit achterwegen laten en gewoon binnen de store module een heldere naam kiezen.

`'Categories/${ADD}'` vind ik niet perse duidelijker dan `'Categories/add'
En waarschijnlijk zorgt dit ervoor dat we alleenmaar die lijst van constants gaan uitbreiden met dingen die maar 1 keer gebruikt worden maar dan wel dynamisch worden. Methods geef je ook geen constants als naam

Maar wat vinden @martn001 en @koenhoeijmakers hier van?

Reply:
Ik vind het een beetje onnodig en het wordt een er onduidelijker van.  Van mij hoeft het niet perse.

Reply:
Because you are not going to use the constants when you are communicating with the store. It's only being used to define the store so they have the 'same' methods, kind of like an interface, but a bit different.

Reply:
Koen is het ook eens, @Deluze verander jij dit?